tests/foreman/ui/test_host.py

```diff
@@ -1535,8 +1535,9 @@ def test_global_registration_upgrade_subscription_manager(
 
 @pytest.mark.tier3
 @pytest.mark.usefixtures('enable_capsule_for_registration')
```

Comment:
we can remove enable_capsule_for_registration ?

Reply:
I'd prefer not to. As it is now, it would fail with:
```
widgetastic_patternfly4.formselect.FormSelectDisabled: FormSelect('.//*[contains(@data-ouia-component-type,"PF4/FormSelect") and @data-ouia-component-id="reg_smart_proxy"]') is not enabled
```
It can be changed so that WebUI doesn't try to fill it but I think it's more future-proof now. Now, `default_smart_proxy` is used which amounts to the Satellite itself but it may change one time and then no fix will be necessary here.